streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -612,6 +713,7 @@ private KafkaStreams getKafkaStreams(final boolean withState,
                 .withCachingEnabled();
 
             builder.addStateStore(storeBuilder);
+            CLUSTER.deleteTopicsAndWait(applicationId + "-" + storeName + "-changelog");
```

Comment:
Because we run the only stateful test twice now, we need to delete the corresponding changelog topic, too.